gammapy/irf/core.py

```diff
@@ -546,6 +554,32 @@ def pad(self, pad_width, axis_name, **kwargs):
             data=data, axes=axes, meta=self.meta.copy(), unit=self.unit
         )
 
+    def data_allclose(self, other, **kwargs):
+        """Compare two data IRFs for equivalency
+
+        Parameters
+        ----------
+        other : the irf to compare against
+            `gammapy.irfs.IRF`
+        kwargs : dict
+                keywords passed to `numpy.allclose`
+                The default tolerance is rtol=1e-3
+
+        Returns
+        -------
+        state : bool
+        """
+
+        kwargs.setdefault("rtol", 1e-3)
+        if self.data.shape != other.data.shape:
+            return False
+        return np.allclose(self.quantity, other.quantity, **kwargs)
+
+    def __eq__(self, other):
+        class_eq = isinstance(other, self.__class__)
+        axes_eq = self.axes == other.axes
+        return bool(class_eq & axes_eq & self.data_allclose(other))
```

Comment:
`&` is a bitwise operator. Here it should work but I think the correct syntax should be:
`class_eq and axes_eq and self.data_allclose(other)`